custom/enikshay/forms.py

```diff
@@ -66,6 +67,7 @@ def __init__(self, *args, **kwargs):
                 crispy.Field('email'),
                 crispy.Field('task'),
                 crispy.Field('case_type'),
```

Comment:
You can remove this field now, Case type is not needed anymore. Its fixed for each command so user need not enter it.